### PR TITLE
fix(feed): setting zoom level when location not provided

### PR DESCRIPTION
**Pull request by sujal-into**
fix: #1367

### CODE REVIEW

File: packages/components/socialFeed/Map/Map.web.tsx [hunk 1]
```diff
@@ -205,8 +206,9 @@ export const Map: FC<MapProps> = ({
         center={
           consultedPostLocation || creatingPostLocation || DEFAULT_MAP_POSITION
         }
-        zoom={12}
+        zoom={!consultedPostLocation && !creatingPostLocation ? 2 : 12}
         attributionControl={false}
+        minZoom={2}
```
Comment (WaDadidou) [inline packages/components/socialFeed/Map/Map.web.tsx:211]: To have the minimal zoom level and avoid white borders
image

```suggestion
        minZoom={2.15}
```

Comment (n0izn0iz) [reply packages/components/socialFeed/Map/Map.web.tsx:211]: this seem like an arbitrary value, we should be able to compute a proper value from map width and height I think

Comment (WaDadidou) [reply packages/components/socialFeed/Map/Map.web.tsx:211]: You're right
`2.15` is ok with our layout, but the zoom could be more optimal on desktop
The goal is "Compute the smallest `zoom` and `minZoom` value to never see the white background"

File: packages/components/socialFeed/Map/Map.web.tsx [hunk 2]
```diff
@@ -197,6 +197,7 @@ export const Map: FC<MapProps> = ({
           width: "100%",
           height: "100%",
           alignSelf: "center",
+          borderRadius: 20,
```
[the remark this answers is not in the document]
Comment (sujal-into) [reply packages/components/socialFeed/Map/Map.web.tsx:200]: If we provide background color to this view, the background color is seen in the edges. The number is just random, any number above 15 is fine. But I will remove this one as this is not related to this issue.

File: packages/components/socialFeed/Map/Map.web.tsx (hunk 1 above)
Comment (sujal-into) [reply packages/components/socialFeed/Map/Map.web.tsx:211]: I will implement this in #1399.

File: packages/components/socialFeed/Map/Map.web.tsx [hunk 3]
```diff
@@ -205,8 +206,9 @@ export const Map: FC<MapProps> = ({
         center={
           consultedPostLocation || creatingPostLocation || DEFAULT_MAP_POSITION
         }
-        zoom={12}
+        zoom={!consultedPostLocation && !creatingPostLocation ? 2 : 12}
```
[the remark this answers is not in the document]
Comment (WaDadidou) [reply packages/components/socialFeed/Map/Map.web.tsx:208]: See https://github.com/TERITORI/teritori-dapp/pull/1399 for a dynamic `zoom` value

File: packages/components/socialFeed/Map/Map.web.tsx (hunk 1 above)
Comment (WaDadidou) [reply packages/components/socialFeed/Map/Map.web.tsx:210]: zoom and bounds are correlated, so ok

**Review by WaDadidou — APPROVED** [review]

**Review by hthieu1110 — APPROVED** [review]
LGTM

**Review by n0izn0iz — CHANGES_REQUESTED** [review]
please use the maximum responsive size instead of an arbitrary zoom level

**Comment by sujal-into** [comment]
> please use the maximum responsive size instead of an arbitrary zoom level

fixed this in this pr: https://github.com/TERITORI/teritori-dapp/pull/1399

**Comment by clegirar** [comment]
Okay so i'll close this PR and add the changes to the #1399, they are so linked, it will more clear.
And even with the changes, it will still an atomic PR 👍

**Comment by clegirar** [comment]
Okay so i'll close this PR and add the changes to the #1399, they are so linked, it will more clear.
And even with the changes, it will still an atomic PR 👍

**Pull request closed without merging**